Modix.Services/IsUp/IsUpService.cs

```diff
@@ -0,0 +1,35 @@
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Discord.Commands;
+using Newtonsoft.Json;
+
+namespace Modix.Services.IsUp
+{
+    public class IsUpService
+    {
+        private const string _apiBaseURl = "https://isitdown.site/api/v3/";
+
+        protected IHttpClientFactory HttpClientFactory { get; }
+
+        public IsUpService(IHttpClientFactory httpClientFactory)
+        {
+            HttpClientFactory = httpClientFactory;
+        }
+
+        public async Task<IsUpResponse> GetIsUpResponseAsync([Summary("Url to get status of")]string url)
```

Comment:
Does `[Summary]` actually do anything for service class methods?

Reply:
no